### PR TITLE
Add jekyll structure

### PR DESCRIPTION
**Pull request by rhynobytes**
# Changes Made

- Updated structure to separate files into 'components' that we can assign to team members and work independently.
- Created SCSS structure for us to work in independent css files with the same names as their include file
- Updated includes to separate this page into semantic sections (nav, section, main, footer, etc.)
- Added google fonts and initial global fonts

## After PR is Merged
Once this review is complete and the changes are merged back into our 'convert-to-jekyll' branch, any of us can branch off of their for our individual tasks.

The structure is all setup to work so that you can run ```jekyll serve``` from your terminal and your changes will update for your browser pointing to http://localhost:4000

### CODE REVIEW

File: _sass/main.scss [hunk 1]
```diff
@@ -0,0 +1,123 @@
+/* Global Styles */
+/* mixins, colors, etc. */
+
+$cfd-red: #E14E54;
+$navy: #181E34;
+$white: white;
+$light-grey: #C4C4C4;
+
+$breakpoints: (
+  'small':  ( min-width:  767px ),
```
Comment (stephencarico) [inline _sass/main.scss:10]: not a big deal, but small width should be 768px https://getflywheel.com/layout/css-breakpoints-responsive-design-how-to/

File: _sass/main.scss [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+html, body {
+  margin: 0;
+  font-family: 'Montserrat', sans-serif;
+}
+
+/* typeography */
+
+h1 {
+  font-family: Montserrat;
```
Comment (stephencarico) [inline _sass/main.scss:41]: Do we need to specify 'Monsterrat' here if this is the default font-family for all elements in html, body? Same goes for the other elements below.

File: _sass/main.scss (hunk 1 above)
Comment (rhynobytes) [reply _sass/main.scss:10]: You're right. I copied that scss variable from somewhere and only played with it to confirm it was being processed correctly. I will update this.

File: _sass/main.scss (hunk 2 above)
Comment (rhynobytes) [reply _sass/main.scss:41]: No, we do not need to re-specify the font family. We will need to clean up the css as we go. I made some comments on the h1 styles that will need to be updated too.

**Pull request merged**